testsupport/tiers/checks.go

```diff
@@ -1786,6 +1788,47 @@ func namespaceManagerSaEditRoleBinding() namespaceObjectsCheck {
 	}
 }
 
+func namespaceManagerSaAdditionalArgocdReadRoleBinding() namespaceObjectsCheck {
+	return func(t *testing.T, ns *corev1.Namespace, memberAwait *wait.MemberAwaitility, owner string) {
+		rb, err := memberAwait.WaitForRoleBinding(t, ns, toolchainv1alpha1.AdminServiceAccountName, toolchainLabelsWaitCriterion(owner)...)
```

Comment:
the third parameter is the name of the RoleBinding
```suggestion
		rb, err := memberAwait.WaitForRoleBinding(t, ns, "additional-argocd-read", toolchainLabelsWaitCriterion(owner)...)
```

Reply:
Oh, that settles it. Thanks!